First code review section

jovasal opened the pull request:
this is my first code review alonde

jovasal committed:
any code for help

any code inside

jovasal committed:
changed

changed

jovasal committed:
change requiered

change requiered

jovasal committed:
errors caused

ok

jovasal commented on index.html:
```diff
@@ -0,0 +1,48 @@
+        <title>finish-mobile-version</title>
+    </head>
+<body>
+    <div class="grid-container">
+        <div class="flex1">
+            <div class="snapshoot">
+                    <img src="./images/Snapshoot Portfolio.png"/>                
+            </div>
+            <div class="tonic">Tonic</div>
+            <div>3</div>
+            <div clas=""><p>A daily selection of privately </br>personalized reads; no accounts or </br>sign-ups required.</p></div>
+            <div>5</div>
+            <div>6</div>
+        </div>
+        <div class="flex2">
+            <div>1</div>
+            <div>2</div>
+            <div>3</div>
+            <div>4</div>
+            <div>5</div>
+            <div>6</div>
+        </div>
+        <div class="flex3">
+            <div>1</div>
+            <div>2</div>
+            <div>3</div>
+            <div>4</div>
+            <div>5</div>
+            <div>6</div>
+        </div>
+        <div class="flex4">
+            <div>1</div>
+            <div>2</div>
+            <div>3</div>
+            <div>4</div>
+            <div>5</div>
+            <div>6</div>
+        </div>
+    </div>
```
Good job using GRID CONTAINER. However, you need to finish your code. in unclear in this time.

jovasal commented on style.css:
```diff
@@ -0,0 +1,102 @@
+  font-size: 30px;
+  justify-content: center;
+  align-content: center;
+}
```
Amazing templates made for your code!. Kindly, could you eliminate blank spaces?

![image](https://github.com/jovasal/first-code-review/assets/35693330/6737ae22-1a6f-4332-93fe-fb749d437baf)

jovasal commented on style.css:
```diff
@@ -0,0 +1,102 @@
+  height: 36;
+  font-family: "Poppins", sans-serif;
+  background-color: #172b4d;
+}
```
Fantastic code. Whereas, is important to delete unnecessary characters:

![image](https://github.com/jovasal/first-code-review/assets/35693330/b78db3fe-ec58-4643-8b6a-8938a3d28e5f)

to

![image](https://github.com/jovasal/first-code-review/assets/35693330/87176770-b12e-4471-bdb9-526e80658623)

jovasal reviewed (commented):
# changes required: 🥥 ♻️ 

https://media.giphy.com/media/V4NSR1NG2p0KeJJyr5/giphy.gif

Please, check my comments below and make all required changes.

Thank you so much

Alusp commented on index.html:
```diff
@@ -0,0 +1,48 @@
+        <title>finish-mobile-version</title>
+    </head>
+<body>
+    <div class="grid-container">
```
- [ ] Please be more clear, and polite in your comments and try to tell why students should follow your advice. It seems like you are ordering the student for something and even your comment is not clear to understand. Kindly use some polite words with some friendly emoji to write a review comment.

### Your Comment

> Good job using GRID CONTAINER. However, you need to finish your code. in unclear at this time.

jovasal replied:
Im really impressed with your advance!! Well done! 🚗 . However, you can finish your code As suggested in the exercise.

you can get more information in:

https://www.w3schools.com/css/css_grid_container.asp

👍 👍 👍 👍 👍 👍 👍 👍

Alusp commented on style.css:
```diff
@@ -0,0 +1,102 @@
+  background-color: #e94a4a;
+  padding: 10px;
+}
+
+
+
+
+
```
- [ ] Please be more, clear, and polite in your comments and try to tell why students should follow your advice like spaces in your code makes code lengthier and difficult to maintain even if it can lead any person in the wrong direction because of confusion.

### Your Comment

> Amazing templates made for your code!. Kindly, could you eliminate blank spaces?

jovasal replied:
Your code is amazing. You are too short to be a Master! 💯 

I suggest reading more information about how to write code clearly. 🐰 Sometimes we forgot some things.

Please, delete all blank spaces that are not necessary.

🍓 💪 🍓

Alusp commented on style.css:
```diff
@@ -0,0 +1,102 @@
+#Tonic {
+  display: flex;
+  width: 295px; hjik
+  height: 36;
```
- [ ] Please be more specific in your comments and try to tell the problem more elaborately so that the student should be able to understand what are you trying to say with a solid reason to make a student a better developer. 

### Your Comment
> Fantastic code. Whereas, is important to delete unnecessary characters:

jovasal replied:
Well done!!! you are too short to do an excellent programmer 👍 👍 

I would kindly suggest deleting hjik in line 56. It might be helpful to consider implementing all code for a better understanding project. 👍


🥇 🥇 🥇 🥇

Alusp reviewed (changes requested):
Hi @jovasal , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![close](https://media.giphy.com/media/t8YPf3ACKeouLzEIvr/giphy.gif)


## Highlights

- [ ] Good PR Title :green_circle: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

stanleySimeon reviewed (changes requested):
# Hi @jovasal, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have all linter checks passed. Kindly, I suggest you fix that before asking for another Review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

jovasal committed:
changes were made but those changes not applied for this excersice,

The focus in this exersice is to explain as a coder reviw a problems.

Also, I made changes under reviewer that dont have idea about this activity.

jovasal committed:
changes

changes

jovasal committed:
changes

changes

jovasal committed:
changes

changes

jovasal committed:
changes

changes

jovasal committed:
changes

changes

jovasal commented:
The idea for this activity is to check my comments as a code review. I don't understand, why my last code review said that linters need to pass this activity because the idea is to show problems in the code. However, linters are passed now without reason and I hope that the next code review understands this situation.

bobb-Rob reviewed (approved):
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @jovasal, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![](https://camo.githubusercontent.com/2c67944c14f5c74b9b607df18d73efb0c423dd148b3e10cb75b445a00220332e/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f5061536e555a573145657970756c734f77642f67697068792e676966)

## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very Good use of inline comments.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
